### ISSUE TITLE
Feature request: “rewards” for taking breaks

### ISSUE THREAD

**Issue by calvarez-ov** [tom-james-watson/breaktimer-app#16]
I often find myself skipping or postponing breaks. I might be motivated to better respect the break time if the app kept score or somehow gave me points/achievements for taking my breaks 😊

I know it sounds silly but it might work.

**Comment by tom-james-watson** [comment]
Yeah I totally agree - this is something I've been thinking about for a long time.

I don't have the time to work on this realistically for the next month, but this is something I'd like to implement in some way or another at some point.

Some potential ideas:

* Change the colour of the tray icon based on how long it's been since you took a break. So, past a certain (configurable?) point, the icon changes to yellow, then eventually to red.
* Show how long it's been since you took a break in the tray menu, or also maybe how many breaks you've taken or skipped.
* Changing the message of the "break about to start" notification. Could include something like the above info. Maybe only once you've skipped or postponed a break?
* Condensing the above info into some kind of "score" that goes up or down based on taking / skipping / postponing breaks. Show that score in the tray menu, notifications or an app window.

**Comment by tom-james-watson** [comment]
Also I've just opened https://github.com/tom-james-watson/breaktimer-app/issues/17, which probably would help in your case too!

**Comment by Satyrars** [comment]
Something akin to 'don't break the streak' principle. But even a simple counter of brakes and skips/postpones would be nice.

**Comment by TKone7** [comment]
I am not sure if it goes into the same direction. But I do set certain "rewards" before I start to work. This can be simple things like, "drink a cup of coffee", "rest in the hammock", "watch a video on Youtube". It would be great to set this in the timer, so you get reminded to redeem your reward.

**Comment by tom-james-watson** [comment]
As of 2.0.0, the upcoming break notification will now guilt-trip you by telling you how long it's been since your last break, if you have recently skipped/snoozed/ended a break early.

I think for now, that's as far as we really need to go for gamification. Open to other ideas, though.
